examples/systemcalls/systemcalls.c

```diff
@@ -9,13 +23,25 @@
 */
 bool do_system(const char *cmd)
 {
-
 /*
  * TODO  add your code here
  *  Call the system() function with the command set in the cmd
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
+    int ret;
+
+    ret = system(cmd);
+
+    // check if system() executed correctly
+    if(-1 == ret)
```

Comment:
@Buff-Chandan what if the command executed by `system` returns something other than -1 but still fails?